### PR TITLE
[#191] Update chromedriver version also for geordi rspec

### PR DESCRIPTION
**Pull request by tecden**

### CODE REVIEW

**Review by codener — COMMENTED** [review]
LGTM. Have you tried it locally and confirmed that it behaves as expected?

**Comment by tecden** [comment]
I have made another fix.
I have also tried it locally, it behaves as expected.

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ### Compatible changes
 * `commit` command: The pivotal tracker story is linked in the description of the commit
+* `rspec` command: When running specs the chromedriver is updated to the current version if necessary
```
Comment (codener) [inline CHANGELOG.md:10]: Please specify "necessary": 
```suggestion
* `rspec` command: chromedriver is updated automatically, if auto-update is configured and the app uses selenium-webdriver
```

**Review by codener — APPROVED** [review]
LGTM!

**Comment by codener** [comment]
Has already been merged as https://github.com/makandra/geordi/commit/74889d7491dac255385e8967040148c42be0fb06.

**Pull request closed without merging**